### ISSUE TITLE
Panic on 34 MiB input file

### ISSUE THREAD

**Issue by vks** [teaearlgraycold/binimage#2]
When running binimage with the defaults on a 34 MiB data file, I'm getting the following error:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Custom(Custom { kind: Other, error: StringError("not enough image data provided") }) }', /checkout/src/libcore/result.rs:906:4
```

**Comment by danthedaniel** [comment]
Can you give me the exact file size in bytes and the command line arguments you used?

**Comment by vks** [comment]
Sure! The size was 35250147 bytes and the command was `binimage test.bin test.png`. I can provide the file if you want.

The backtrace:

```
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
             at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at /checkout/src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at /checkout/src/libstd/sys_common/backtrace.rs:60
             at /checkout/src/libstd/panicking.rs:381
   3: std::panicking::default_hook
             at /checkout/src/libstd/panicking.rs:397
   4: std::panicking::rust_panic_with_hook
             at /checkout/src/libstd/panicking.rs:611
   5: std::panicking::begin_panic
             at /checkout/src/libstd/panicking.rs:572
   6: std::panicking::begin_panic_fmt
             at /checkout/src/libstd/panicking.rs:522
   7: rust_begin_unwind
             at /checkout/src/libstd/panicking.rs:498
   8: core::panicking::panic_fmt
             at /checkout/src/libcore/panicking.rs:71
   9: core::result::unwrap_failed
             at /checkout/src/libcore/macros.rs:41
  10: <core::result::Result<T, E>>::unwrap
             at /checkout/src/libcore/result.rs:772
  11: binimage::render_file
             at src/main.rs:115
  12: binimage::main
             at src/main.rs:47
  13: __rust_maybe_catch_panic
             at /checkout/src/libpanic_unwind/lib.rs:99
  14: std::rt::lang_start
             at /checkout/src/libstd/panicking.rs:459
             at /checkout/src/libstd/panic.rs:361
             at /checkout/src/libstd/rt.rs:61
  15: main
  16: __libc_start_main
  17: _start
```

**Comment by danthedaniel** [comment]
This should be fixed as of a3765f242cb50278d141c5d1bb41dca5818b7fee. Can you confirm?

**Comment by vks** [comment]
Thanks for the quick fix, it works now.
